code/modules/mob/living/carbon/human/human.dm

```diff
@@ -66,12 +66,15 @@
 /mob/living/carbon/human/golem/New(var/new_loc, delay_ready_dna = 0)
 	h_style = "Bald"
 	..(new_loc, "Golem")
-	gender = NEUTER
```

Comment:
Slime people have neutral gender, so I needed to make a `gender` species var so that I could set the gender in `set_species()` when someone's race got changed to slime person. Since I added the var, I figured there was no reason not to remove this from golems' `New()` and instead put it in their species datum.